src/main/java/uk/gov/hmcts/juror/api/moj/controller/response/juror/JurorPaymentsResponseDto.java

```diff
@@ -0,0 +1,108 @@
+package uk.gov.hmcts.juror.api.moj.controller.response.juror;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import uk.gov.hmcts.juror.api.validation.ValidationConstants;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.List;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@Builder
+@ToString
+@Schema(description = "Juror appearance and payments log DTO")
+public class JurorPaymentsResponseDto {
+    @JsonProperty("payment_days")
+    @Schema(description = "List of appearance and payments per day")
+    private List<PaymentDayDto> data;
+
+    @JsonProperty("attendances")
+    @Schema(description = "Number of attendances")
+    private long attendances;
```

Comment:
should this be an int rather than a long?

Reply:
Fixed